### PR TITLE
Add documentation

### LINKED ISSUE #10: Add docstrings and expand documentation
Document the basic types such as `SecureVector` or `PlainVector`, including their constructors, and how to use them. Also document auxiliary functions such as `level`, `capacity` etc.

### PR DESCRIPTION
**Pull request by ArseniyKholod**
closes #10 and a bit more.
I added some documentation.

### CODE REVIEW

**Comment by sloede** [comment]
Is this ready for review yet?

**Comment by ArseniyKholod** [comment]
> Is this ready for review yet?

Yes, it is!

File: src/unencrypted.jl [hunk 1]
```diff
@@ -1,15 +1,66 @@
...
+"""
+    init_multiplication!(context::SecureContext{<:Unencrypted}, private_key::PrivateKey)
+
+An empty duplicate of [`init_multiplication!`](@ref) for unencrypted data.
+
+See also: [`SecureContext`](@ref), [`Unencrypted`](@ref), [`PrivateKey`](@ref),
+[`init_multiplication!`](@ref)
+"""
 init_multiplication!(context::SecureContext{<:Unencrypted}, private_key::PrivateKey) = nothing
+"""
```
Comment (sloede) [inline src/unencrypted.jl:34]:
```suggestion
init_multiplication!(context::SecureContext{<:Unencrypted}, private_key::PrivateKey) = nothing

"""
```

File: src/unencrypted.jl [hunk 2]
```diff
@@ -1,15 +1,66 @@
...
+See also: [`SecureContext`](@ref), [`OpenFHEBackend`](@ref)
+"""
 struct Unencrypted <: AbstractCryptoBackend
     # No data fields required
 end
 
+"""
+    generate_keys(context::SecureContext{<:Unencrypted})
+
+Return public and private keys initialized as `nothing` for unencrypted data.
```
[the remark this answers is not in the document]
Comment (ArseniyKholod) [reply src/unencrypted.jl:16]: Thanks! I'll be more careful

**Review by sloede — APPROVED** [review]
LGTM - thanks!

**Comment by sloede** [comment]
Test failures due to upstream issues :cry

**Pull request merged**